packages/web-twig/src/Resources/components/Select/Select.stories.twig

```diff
@@ -0,0 +1,23 @@
+{% extends 'layout/plain.html.twig' %}
+
+{% block content %}
+
+    {# Select Default #}
+    {% include '@components/Select/stories/Select.twig' %}
+
+    {# Select Disabled #}
+    {% include '@components/Select/stories/SelectDisabled.twig' %}
+
+    {# Select Fluid #}
+    {% include '@components/Select/stories/SelectFluid.twig' %}
+
+    {# Select Helper Text #}
+    {% include '@components/Select/stories/SelectHelperText.twig' %}
+
+    {# Select Label Hidden #}
+    {% include '@components/Select/stories/SelectLabelHidden.twig' %}
+
+    {# Select Validation State #}
+    {% include '@components/Select/stories/SelectValidationState.twig' %}
```

Comment:
it would be nice, if it could be sorted as in Web. Because then we would be able to compare it visually.

Reply:
I do not agree, I am based on the examples that are required for TextField and there is no required, it is part of the validation states. Furthermore, I don't have one specifically for Placeholder, but I have placeholders in all examples. It would be good if it were the same for the website, or am I wrong? @crishpeen @literat @adamkudrna

Reply:
TBH examples of all form components except FileUploader are a mess currently. We added new examples (eg. AutoGrow, Password Toggle, ...) and now it is poorly organized. We should separate items into sections, add titles etc. It is already done in the WEB package in Select. And AFAIK we want to have the example in Twig (and React Demo) the same. Otherwise, we won't be able to test it properly using the visual tests. 

Placeholder in Select is specific, so IMHO it makes sense to showcase in a separate section. 

If we decide to change the demo here, in Twig, we should update the Web demo too.

Reply:
The placeholder for select is specific only from the point of view of implementation, but as a form element it should always have a placeholder.

The examples of how they are prepared for Storybook and React demo are the same and arranged in the same way, the same is true for Twig.